_aws.yml

```diff
@@ -1,22 +0,0 @@
-all:
-  hosts:
-    node_172_31_14_52:
-      ansible_host: 172.31.14.52
-    node_172_31_1_13:
-      ansible_host: 172.31.1.13
-    node_172_31_1_238:
-      ansible_host: 172.31.1.238
-    node_172_31_1_86:
-      ansible_host: 172.31.1.86
-    node_172_31_2_161:
-      ansible_host: 172.31.2.161
-    node_172_31_2_233:
-      ansible_host: 172.31.2.233
-    node_172_31_2_246:
-      ansible_host: 172.31.2.246
-    node_172_31_7_131:
-      ansible_host: 172.31.7.131
-  vars:
-    ansible_host_key_checking: false
-    ansible_ssh_private_key_file: /home/ubuntu/.aws/login-us-west-2.pem
-    ansible_user: ubuntu
```

Comment:
I think it's because I ran some ansible commands from this directory. This repo doesn't ignore ymls and I accidentally used `git add .`. It's been deleted now though.